cosmocrat/osm_tools/osmupdate.py

```diff
@@ -0,0 +1,58 @@
+import os
+import cosmocrat.definitions as definitions
+
+from uuid import uuid4
+from cosmocrat.helper_functions import run_command_wrapper
+from cosmocrat.osm_tools.osmconvert import get_osm_file_timestamp
+
+SUBPROCESS_NAME='osmupdate'
+
+def limit_time_units(time_units=definitions.TIME_UNITS_IN_USE):
+    result = ''
+    for time_unit in time_units:
+        if time_unit not in definitions.Time_Unit._member_names_:
+            continue
+        result += f'--{time_unit} '
+    return result
+    
+def get_changes_from_timestamp(input_timestamp, changes_format=definitions.FORMATS_MAP['OSC']):
+    temp_output_name = f'{uuid4()}.{changes_format}'
+    output_path = os.path.join(definitions.OSMCHANGES_PATH, changes_format, temp_output_name)
+
+    run_command_wrapper(f'{definitions.OSMUPDATE_PATH} \
+                    {input_timestamp} \
+                    {output_path} \
+                    {limit_time_units()} \
+                    --tempfiles={definitions.OSMUPDATE_CACHE_PATH} \
+                    --keep-tempfiles \
+                    --trust-tempfiles \
+                    -v',
+                    subprocess_name=SUBPROCESS_NAME)
+
+    output_timestamp = get_osm_file_timestamp(output_path)
+    output_name = f'{input_timestamp}.{output_timestamp}.{changes_format}'
+    new_output_path = os.path.join(definitions.OSMCHANGES_PATH, changes_format, output_name)
+    os.rename(output_path, new_output_path)
+    return new_output_path
+
+def get_changes_from_file(input_path, changes_format=definitions.FORMATS_MAP['OSC']):
+    temp_output_name = f'{uuid4()}.{changes_format}'
+
+    output_path = os.path.join(definitions.OSMCHANGES_PATH, changes_format, temp_output_name)
+    
+    run_command_wrapper(f'{definitions.OSMUPDATE_PATH} \
+                    {input_path} \
+                    {output_path} \
+                    {limit_time_units()} \
+                    --base-url=http://planet.fosm.org/planet/ --base-url-suffix=-replicate \
```

Comment:
does it stop us from using the cli in a closed network?

Reply:
My bad base-url was hardcoded only for testing.
Now the update function gets its base-url as an argument (--source). By default osm-update uses the global osm replications url.
So in closed networks, we'll have to provide a valid inner url.

Reply:
We should update the README.md, didn't see the update in docs

Reply:
look again:
https://github.com/MapColonies/cosmocrat-cli/blob/b811b49c3b3c8108cb28e3762f4557ef7bf6dc4f/README.md#L34-L41